composer.json

```diff
@@ -19,5 +19,14 @@
             "name": "The SilverStripe Community",
             "homepage": "http://silverstripe.org"
         }
+    ],
+    "require-dev": {
+        "silverstripe/markdown-php-codesniffer": "dev-pulls/1/new-repo"
```

Comment:
Will need to update this to use a non fork branch

Reply:
Yup, will do that as soon as the PR for that has been merged and the package is added to packagist.

Reply:
Done